### PR TITLE
distribute fractions normally

### PR DESCRIPTION
**Pull request by ldworkin**
Not sure why we had special logic for fractions before.

Previously:
4(5/4 + x) => (5 * 4)/4 + x * 4

Now:
4(5/4 + x) => 4 * (5/4) + 4 * x

### CODE REVIEW

**Comment by evykassirer** [comment]
omg lol I think I know why this was the case - remember how the cancelling didn't work that well before? I think this was to get around that (not totally sure but that's my guess for now)

File: test/simplifyExpression/distributeSearch/distributeSearch.test.js [hunk 1]
```diff
@@ -53,32 +53,31 @@ describe('distribute', function () {
...
-        '((1 * (2x + x^2)) / (x^2 - 4) + (8x^3 + 4x^4))']
+    ['(2x + x^2) * (1/x + 4x)',
+      ['(2x * (1 / x + 4x) + x^2 * (1 / x + 4x))',
+        '((2x * 1 / x + 8x^2) + (x^2 * 1 / x + 4x^3))']
     ],
 
-    ['(2x + x^2) * (3x^2 / (x^2 -4) + 4x^2)',
-      ['((3x^2 * (2x + x^2)) / (x^2 - 4) + 4x^2 * (2x + x^2))',
-        '((3x^2 * (2x + x^2)) / (x^2 - 4) + (8x^3 + 4x^4))']
+    ['(x - 4) * ((3x^2)/ (x^2 - 4) + 1)',
```
Comment (evykassirer) [inline test/simplifyExpression/distributeSearch/distributeSearch.test.js]: is there a reason why you changed these tests?

Comment (ldworkin) [inline test/simplifyExpression/distributeSearch/distributeSearch.test.js]: Well the old ones were failing -- the answers were equivalent but just came out looking different due to the change. I tried fixing them, but they were pretty complicated and it was hard to follow, so I replaced them with some slightly simpler ones, so it's easier to see what's going on.

**Comment by ldworkin** [comment]
Make sense re: cancelling ... I think?! Though the change I made to the cancelling code was to make it work for parens, and there are no parens here ... anyway hopefully this doesn't break anything terribly! I tested a bunch and Gen will do so more.
